docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+web:
+  build: .
+  ports:
+    - "4000:4000"
+  command: mix phoenix.server
+  environment:
+    - MIX_ENV=dev
+    - PORT=4000
+  volumes:
+    - .:/app
+  working_dir: /app
+  links:
+    - db
+db:
+  image: postgres:9.5
+  environment:
+    - POSTGRES_USER=postgres
```

Comment:
I'm not familiar with docker-compose, but should this be `DB_ENV_POSTGRES_USER` like you had above?